ui/apps/platform/src/Components/ReportJob/MyLastJobStatusTh.tsx

```diff
@@ -10,26 +10,42 @@ function MyLastJobStatusTh() {
                 <Flex direction={{ default: 'column' }} spaceItems={{ default: 'spaceItemsMd' }}>
                     <FlexItem>
                         <p>
-                            The status of your last requested job from the{' '}
-                            <strong>active job queue</strong>. An <strong>active job queue</strong>{' '}
-                            includes any requested job with the status of <strong>preparing</strong>{' '}
-                            or <strong>waiting</strong> until completed.
+                            Displays the status of your most recent job, whether it is currently
+                            running or has completed. The possible statuses are:
                         </p>
                     </FlexItem>
+                    <FlexItem>
+                        <p>
+                            <strong>Waiting:</strong>
+                        </p>
+                        <p>Your job is in the queue, waiting for other jobs to be processed.</p>
```

Comment:
What is pro and con or possessive pronoun here versus definite article elsewhere?
* Your job
* The report

> The report is ready and available for download.

Besides inconsistent, I wonder if after establishing that the subject is your report jobs, neutral wording explains status that might just as well apply to report jobs in general?

Reply:
➕